paimon-hive/paimon-hive-catalog/src/main/java/org/apache/paimon/hive/HiveCatalog.java

```diff
@@ -567,12 +569,24 @@ public static HiveConf createHiveConf(
             if (!hiveSite.toUri().isAbsolute()) {
                 hiveSite = new org.apache.hadoop.fs.Path(new File(hiveSite.toString()).toURI());
             }
+            try (InputStream inputStream = hiveSite.getFileSystem(hadoopConf).open(hiveSite)) {
```

Comment:
Why add this? Remote file is not work in previous code?

Reply:
in previous code if hive-site.xml is illegal，such as xml file format is incorrect it will not throw  execption